Options.py

```diff
@@ -486,6 +486,30 @@ def __ne__(self, other):
         else:
             raise TypeError(f"Can't compare {self.__class__.__name__} with {other.__class__.__name__}")
 
+    def __lt__(self, other: typing.Union[Choice, int, str]):
+        if isinstance(other, str):
+            assert other in self.options, f"compared against an unknown string. {self} < {other}"
+            other = self.options[other]
+        return super(Choice, self).__lt__(other)
+
+    def __gt__(self, other: typing.Union[Choice, int, str]):
+        if isinstance(other, str):
+            assert other in self.options, f"compared against an unknown string. {self} < {other}"
```

Comment:
```suggestion
            assert other in self.options, f"compared against an unknown string. {self} > {other}"
```
is this intention for the assert message? because all of your assert messages are using `<`

Reply:
Lmao copypasta is tasty